frontend/src/components/StatusTags.vue

```diff
@@ -35,11 +46,55 @@ export default {
   },
   mixins: [shootItem],
   computed: {
-    filteredConditions () {
+    ...mapState([
+      'conditionCache'
+    ]),
+    conditions () {
       if (isEmpty(this.shootConditions)) {
         return []
       }
-      return filter(this.shootConditions, condition => !!condition.lastTransitionTime)
+
+      const shootConditions = filter(this.shootConditions, condition => !!condition.lastTransitionTime)
+      const conditions = map(shootConditions, condition => {
+        const { lastTransitionTime, message, status, type, codes } = condition
+        const id = type
+        const { displayName: name, shortName, description, showAdminOnly } = this.conditionMetadataFromType(condition.type)
+
+        return { id, name, shortName, description, message, lastTransitionTime, status, codes, showAdminOnly }
+      })
+
+      return sortBy(conditions, 'shortName')
+    }
+  },
+  methods: {
+    ...mapMutations([
+      'setCondition'
+    ]),
+    conditionMetadataFromType (type) {
```

Comment:
Conditions do not have metadata. Metadata are data that describe the data. The name pattern you are using here is a pattern that we have introduced for getters that return a function with parameters. But here we have a method. So the name should be `getCondition`.

Reply:
renamed to `getCondition`